README.md

```diff
@@ -2,40 +2,116 @@
 
-Rack API server
+### Private and Secure
 
-### [client](https://github.com/convox/rack/tree/master/client)
+Rack runs in an isolated VPC that only you and your team have access to. Application builds take place in a single-tenant build service and the resulting Docker images are stored in a private ECS Container Registry. Application secrets are stored in S3, encrypted with KMS, a hardware security module. Application logs are archived in CloudWatch LogsGroups.
 
-Rack API client
+Your network is isolated, your platform is single-tenant, and your application data never leaves your AWS account.
 
-### [cmd/agent](https://github.com/convox/rack/tree/master/cmd/agent)
+### Simple and Reliable
 
-Convox agent for ECS instances
+Apps run as Docker containers on ECS with HTTP access through ELBs. This architecture is modern, simple and provably reliable and scalable.
 
-### [cmd/convox](https://github.com/convox/rack/tree/master/cmd/convox)
+Container Logs are extracted from Docker with its native APIs and log drivers. Docker daemon options are minimally changed to avoid observed log rotation problems.
 
-Convox CLI
+Logs are stored in CloudWatch Logs for archival and search, and Kinesis for streaming. This is simple and cost effective for any volume of logs.
 
-### [cmd/crypt](https://github.com/convox/rack/tree/master/cmd/crypt)
+Complex and experimental things like overlay networking, persistent container volumes, and distributed file systems are simply not supported at the moment.
 
-Encrypt/decrypt sensitive information
+All throughout the stack we aim to leverage managed services and mature systems to accomplish tasks at hand. AWS offers the vast majority of infrastructure services and Docker the vast majority of runtime functionality.
 
-## Test
+### Easy to Maintain
+
+Platform updates are automatically applied with the `convox rack update` command. U
+
+Updates are executed with CloudFormation, so you can be confident that they will be safely executed.
+
+Some updates are simple Rack API changes that will roll out in seconds.
+
+Some updates are base security updates like a new AMI, Linux Kernel, or Docker engine. These are rolled out one instance at a time and are guaranteed to not cause application downtime.
+
+Some updates are infrastructure migrations. For example, ECR is still in limited availability, not yet available in eu-west-1. When it does become available there, a future `convox rack update` will safely migrate your EU clouds over to it.
+
+### Open Source
+
+Rack is open source and free (as in beer and in speech) to use. You can look at the source code to audit how it configures your AWS account. You you fork it and modify. You can contribute your ideas and patches back to the project so we can all share.
+
+### Philosophy
+
+The Convox team and the Rack project have a strong philosophy about how to manage cloud services. Some choices we frequently consider:
```

Comment:
Yeah I'd like to link all these to blog posts or other docs which have yet to be written. Is it confusing having these here without supporting context? I'm thinking about dropping this section for now.